src/IIIFPresentation/API/Program.cs

```diff
@@ -54,6 +55,8 @@
 builder.Services.AddCaching(cacheSettings);
 builder.Services
     .AddSingleton<IETagManager, ETagManager>()
+    .AddSingleton<IPathGenerator, PathGenerator>()
+    .AddSingleton<IHttpContextAccessor, HttpContextAccessor>()
```

Comment:
There's a framework method for this `.AddHttpContextAccessor()`. I suspect it does the same as this under the hood but might be worth changing so that if `AddHttpContextAccessor` in dotnet10 does something different we get that behaviour